listing.php

```diff
@@ -10,18 +10,19 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="normalize" href="app/css/normalize.css"/>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
     <link rel="stylesheet" href="app/css/style.css"/>
     <title>Solid Properties</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body>
 <div class="header">
     <div class="basicNavbar">
         <div class="basicNavbarButtons">
-            <button id="allBtn">All</button>
-            <button id="salesBtn">Sales</button>
-            <button id="lettingsBtn">Lettings</button>
+            <a href="index.php?type=all"><button class="allBtn navButtons" >All</button></a>
+            <a href="index.php?type=sales"><button class="salesBtn navButtons">Sales</button></a>
```

Comment:
type=1 instead of sales and type=2 instead of lettings as we will be using those types to show content.